academy/ibc-dev/ibcapp_packet.md

```diff
@@ -93,36 +93,40 @@ message MsgSendIbcTopRankResponse {
 ```
 <HighlightBox type="info">
 
-The proto message `MsgSendIbcTopRank` includes the field `timeoutTimestamp` which is added by Ignite CLI when scaffolding an IBC packet. It is an IBC channel parameter that is important in IBC and Ignite CLI abstracts this away from the user to have to add this manually.
+The proto message `MsgSendIbcTopRank` includes the field `timeoutTimestamp`, which is added by Ignite CLI when scaffolding an IBC packet. This is an IBC channel parameter that is important in IBC and Ignite CLI abstracts this away, making it necessary for the user to add this manually.
```

Comment:
```diff
- This is an IBC channel parameter that is important in IBC and Ignite CLI abstracts this away, **making it necessary** for the user to add this manually.
+ This is an IBC channel parameter that is important in IBC and Ignite CLI abstracts this away, **removing the need** for the user to add this manually.
```

Reply:
Ah, thank you, I misinterpreted the original completely.